spec/features/view_order_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+feature "can view profile page via link on order show page" do
+
+  scenario "user clicks profile link" do
+    user = User.create(first_name: "Donald",
+                       last_name: "Trump",
+                       email: "trump@luxury.com",
+                       username: "therealtrump",
+                       password: "password",
+                       password_confirmation: "password")
+
+    candidate = Candidate.create(name: "Ted Cruz", party: "Republican", bio: "Kim Davis", last_name: "cruz")
+    issue = Issue.create(topic: "Gun Control", description: "Guns Guns Guns!", picture: "guns")
+    candidate_issue = CandidateIssue.create(candidate: candidate, issue: issue, stance: "Give them the guns!")
```

Comment:
Useless assignment to variable - `candidate_issue`. Did you mean `candidate`?<br>Line is too long. [110/80]